cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -3,16 +3,17 @@ NAME
                         definition
 
 SYNOPSIS
-       bal grpc [--input <proto-file-path>] [--output <path>]
-                        [--mode] (client | service | proxy)
+       bal grpc [--input] <proto-file-path> [--output] <path>
+                        [--mode] (client | service)
```

Comment:
```suggestion
                        [--mode mode ]
```

Reply:
You can specify the mode values in the option description

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/c041ce5fcdfe872fc4378580d5d0b975ecbde922